### ISSUE TITLE
What do you mean by same mel-spectogram representation?

### ISSUE THREAD

**Issue by williamFalcon** [NVIDIA/tacotron2#99]
Will this work if:
1. I train the tacotron2 model here using the settings in the repo.
2. Use the pretrained weights from the waveglow repo?

How can I verify the settings for the mel-spectogram are the same?

**Comment by rafaelvalle** [comment]
The defaults here are the same as the defaults in waveglow repo.
You're looking at these params: https://github.com/NVIDIA/tacotron2/blob/master/hparams.py#L35

**Comment by williamFalcon** [comment]
ok awesome. So just the sampling rate has to be the same? Or do you mean the whole audio section?


Btw I generated the following sample (i only trained  it for 20 epochs) i assume this has to go for something closer to 500 epochs?
https://www.dropbox.com/sh/jlrfrucpd1riisj/AAB5WbLNTVn3bm5N86JgR4Lva?dl=0

**Comment by rafaelvalle** [comment]
```

        sampling_rate=22050,
        filter_length=1024,
        hop_length=256,
        win_length=1024,
        n_mel_channels=80,
        mel_fmin=0.0,
        mel_fmax=8000.0,
```

**Comment by rafaelvalle** [comment]
For tacotron 2 with the current setup and mel representation a validation loss of ~3 and below sounds like speech for sure.
